src/app/containers/LiquidityMining/index.tsx

```diff
@@ -73,21 +74,52 @@ export function LiquidityMining() {
     return output;
   });
 
-  const rows = combinedData.map((item, key) => {
-    if (getContractNameByAddress(item.pool)?.includes('USDT')) {
-      return <PoolData key={key} data={item} isConnected={isConnected} />;
-    } else {
-      return null;
-    }
-  });
+  const USDTData = combinedData.find(
+    item =>
+      getContractNameByAddress(item.pool)?.includes('USDT') &&
+      symbolByTokenAddress(item.asset).includes('USDT'),
+  );
+
+  const BTCData = combinedData.find(
+    item =>
+      getContractNameByAddress(item.pool)?.includes('BTC') &&
+      symbolByTokenAddress(item.asset).includes('BTC'),
+  );
 
   return (
     <div className="">
       <h1 className="w-100 text-center mb-3">
         Liquidity Mining: USDT/BTC Pool
       </h1>
       <div className="d-flex flex-wrap mb-5"></div>
-      <div className="d-flex flex-wrap justify-content-around">{rows}</div>
+      <div className="d-flex flex-wrap justify-content-around">
+        <PoolData
+          data={
+            USDTData || {
+              asset: getContract('USDT_token').address,
+              pool: getContract('USDT_amm').address,
```

Comment:
Would recommend to use getTokenContract and getAmmContract helpers instead, in case name properties would change here.